blocks/gdplus-subscribe-hero/gdplus-subscribe-hero.js

```diff
@@ -0,0 +1,149 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+import { parseFragment, removeEmptyElements, render } from '../../scripts/scripts.js';
+
+const offerCards = [
```

Comment:
I think this kind of data should be editable by non developers so maybe you can put it in a sheet and just fetch the sheet from here ? Same for benefits.

Reply:
I added the card and benefit data to the custom-data spreadsheet. I used async await because I didn't have a lot of time to implement a refactor for `window.store` and its hard to display anything on the page without this data.
I can change this next week if you prefer.

Reply:
No it's fine since it's only for this page 👍